frontend/web/components/modals/CreateFlag.js

```diff
@@ -555,6 +617,73 @@ const CreateFlag = class extends Component {
             />
           </FormGroup>
         )}
+        {metadataEnable && (
+          <FormGroup className='mb-5 setting'>
+            <InputGroup
+              title={'Metadata'}
+              tooltip={`${Constants.strings.TOOLTIP_METADATA_DESCRIPTION} flags`}
+              tooltipPlace='left'
+              component={
+                <div>
+                  <Row>
```

Comment:
It looks like this renders out the defined metadata.

- This should be componentised and reused.
- Does this inform you of any missing required metadata?

Reply:
I'll check it, thanks for the suggestion

Reply:
A New Component was added